frontend/src/index.pug

```diff
@@ -170,10 +170,15 @@ html
                   title="click to view author's repo"
                 )
                   i.summary-chart__title--button.fab.fa-github
-
+                a(
+                  title="click to view breakdown of commits",
+                  onclick="deactivateAllOverlays()",
+                  v-on:click="openTabZoom(user, filterSinceDate, filterUntilDate)"
+                )
+                  i.summary-chart__title--button.fas.fa-list-ul
               .summary-chart__ramp(
                 onclick="viewClick(event)",
-                v-on:click="openTabZoom(user)"
+                v-on:click="getTabZoomRange(user)"
```

Comment:
Maybe consider to rename it to `getRangeAndOpenTabZoom`? `getTabZoomRange` is not accurate enough.